sources/packages/backend/apps/api/src/services/application-offering-change-request/application-offering-change-request.service.ts

```diff
@@ -100,4 +103,73 @@ export class ApplicationOfferingChangeRequestService {
       count,
     };
   }
+
+  /**
+   * Gets application offering request change list.
+   * @param locationId location id.
+   * @param paginationOptions options to execute the pagination.
+   * @param statuses list of status that need to be included in the query.
+   * @returns list of requested application offering changes.
+   */
+  async getRequestedSummary(
+    locationId: number,
+    paginationOptions: PaginationOptions,
+    statuses: ApplicationOfferingChangeRequestStatus[],
+  ): Promise<PaginatedResults<ApplicationOfferingChangeRequest>> {
+    const offeringChange = this.applicationOfferingChangeRequestRepo
+      .createQueryBuilder("applicationOfferingChangeRequest")
+      .select([
+        "applicationOfferingChangeRequest.id",
+        "applicationOfferingChangeRequest.applicationOfferingChangeRequestStatus",
+        "application.applicationNumber",
+        "currentAssessment.id",
+        "offering.studyStartDate",
+        "offering.studyEndDate",
+        "student.id",
+        "user.firstName",
+        "user.lastName",
+        "applicationOfferingChangeRequest.assessedDate",
```

Comment:
Minor, all the applicationOfferingChangeRequest selects can be put to gether. 

i.e. line 131 132 right below 123